src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -7106,7 +7108,7 @@ export interface OfficeAPI {
    * @param size - Amount of positions to open
    * @returns Cost of upgrading the office
    */
-  getOfficeSizeUpgradeCost(divisionName: string, city: CityName | `${CityName}`, asize: number): number;
+  getOfficeSizeUpgradeCost(divisionName: string, city: CityName | `${CityName}`, size: number): number;
```

Comment:
need to `npm run doc` to get this updated in the relevant docs.

Reply:
I'm a bit hesitant to change this from a one-file PR to a 25-file PR. I think that doc command is part of cutting a release, but if the maintainers want I can run that for this PR.

Reply:
+1 on it being part of the intended process. Really, those files should be autogenerated as opposed to checked-in, but there are issues with doing that at this point.